java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -485,6 +485,51 @@ public void throwWrongInputType() throws OrtException {
     }
   }
 
+  @Test
+  public void throwWrongSizeInput() throws OrtException {
+    SqueezeNetTuple tuple = openSessionSqueezeNet();
+    try (OrtSession session = tuple.session) {
+
+      float[] inputData = tuple.inputData;
+      NodeInfo inputMeta = session.getInputInfo().values().iterator().next();
+      Map<String, OnnxTensor> container = new HashMap<>();
+      float[] wrongSizeData = Arrays.copyOf(inputData, 2 * 224 * 224);
+      Object tensor = OrtUtil.reshape(wrongSizeData, new long[] {1, 2, 224, 224});
+      container.put(inputMeta.getName(), OnnxTensor.createTensor(env, tensor));
+      try {
+        session.run(container);
+        OnnxValue.close(container.values());
```

Comment:
> OnnxValue.close(container.values()); [](http://example.com/codeflow?start=7&length=37)

Should this not be in a finally clause? The same below in the next test

Reply:
Fixed.